src/resources/elements/primeDesignSystem/pform-input/pform-input.html

```diff
@@ -2,14 +2,10 @@
 <label class="formInputLabel" if.bind="label">
   <span class="formInputLabelTitle">
     ${label}
-    <pquestion-mark
-      class="labelInfoIcon"
-      if.bind="labelInfo"
-      text.bind="labelInfo"
-    ></pquestion-mark>
+    <pquestion-mark class="labelInfoIcon" if.bind="labelInfo" text.bind="labelInfo"></pquestion-mark>
   </span>
   <div class="formInputLabelExtra" if.bind="showCounter">
-    ${inputReference.value.length || 0}${maxLength ? '/' + maxLength : ''}
+    ${charValue || inputReference.value.length || 0}${maxLength ? '/' + maxLength : ''}
```

Comment:
Please note that you're changing a design-system element here. Though integrated with this project, it might be used as a reference in other projects. Is this change applied to the DS or is it specific to the clause text area? If the latter- it may be best to implement it outside of the DS and make sure it's not implicating any of the currently used elements...

@hiaux0 - what's your opinion here?

Reply:
my idea was to extend, which means other instances shouldn't be affected. if you don't pass `charValue` or  `isVertical`, then nothing happens. but if you are unconvinced, by implementing it outside of DS, you mean just making a separate component specifically for the clause fnality ?